release-content/0.16/release-notes/17398_Relationships_nonfragmenting_onetomany.md

```diff
@@ -1,4 +1,63 @@
-<!-- Relationships (non-fragmenting, one-to-many) -->
-<!-- https://github.com/bevyengine/bevy/pull/17398 -->
+When building Bevy apps, it is often useful to "link" entities together. The most common case in Bevy is connecting parent and child entities together. In previous Bevy versions, a child would have a `Parent` component, which stored a reference to the parent entity, and the parent entity would have a `Children` component, which stored a list of all of its children entities. To ensure these connections remained valid, developers were not allowed to modify these components directly. Instead, all changes had to be done via specialized commands.
 
-<!-- TODO -->
+This worked, but it had some pretty glaring downsides:
+
+1. Maintaining hierarchies was "separate" from the core ECS data model. This made it hard to improve our spawn APIs, and made interacting with hierarchies less natural.
+2. The system was specialized and not reusable. Developers wanting to define their own relationship types had to reinvent the wheel.
+3. To ensure data integrity, expensive scans were required to avoid duplicates.
+
+In **Bevy 0.16** we have added initial support for Relationships: a generalized and efficient component-driven system for linking entities together bidirectionally. This is what defining a new [`Relationship`] looks like:
```

Comment:
I prefer the use of bold to define new terms. The capitalization here initially looks like a mistake.